pallets/ocex/src/lib.rs

```diff
@@ -213,6 +217,21 @@ decl_module! {
             Ok(())
         }
 
+        /// Uploads CID for given Enclave id
+        ///
+        /// # Parameter
+        ///
+        /// * `new_cid`: CID to be uploaded for given Enclave Id
+        #[weight = 10000]
+        pub fn upload_cid(origin, new_cid: T::Hash) -> DispatchResult{
```

Comment:
Why are you using T::Hash for the cid?

Reply:
maybe check with Allan what intends to use. In the worker I've started to use the cid from crates.io. Not sure if this is the optimal solution though.

Reply:
In issue it was mentioned as T::Hash 😅

Reply:
Sorry, my bad ! Can you check with Allan?

Reply:
Is the size of the CID fixed in size and the same length as T::Hash? Then it will work but it is wrong and it is fragile, we should add a CID type internally insterad.

Reply:
Definitely will be using the CID type, importing it like @haerdib mentioned.